### PR TITLE
Add video for Alibaba environment

### PR DESCRIPTION
**Pull request by AndrewJakubowicz**
# Issue

We can't play the current video on our china domain.

# Fix

Self host the video just for the china version of the homepage.

### CODE REVIEW

File: app/models/User.coffee [hunk 1]
```diff
@@ -522,6 +522,7 @@ module.exports = class User extends CocoModel
   hideFooter: -> @isTarena()
   useGoogleClassroom: -> not (features?.chinaUx ? false) and me.get('gplusID')?   # if signed in using google SSO
   useGoogleAnalytics: -> not (features?.chinaInfra ? false)
+  isCocoChinaInfra: -> features?.chinaInfra ? false
```
Comment (shubhi1092) [inline app/models/User.coffee]: Should we rename this flag to be specific to video hosting?
`isCocoChinaInfra` seems a duplicate of `features.chinaInfra`

**Comment by AndrewJakubowicz** [comment]
@shubhi1092 Thanks for the code review.

My thought process was to keep this separate from what netease is using.
`isCocoChinaInfra` should point to a global feature flag `cocoChinaInfra` that is set on this environment. I can add this flag to the database and rename it here.

My logic for having this be a single flag is that there will be many changes that will have to happen under the `cocoChinaInfra` feature flag.

What do you think?

Should I add a new global feature flag `features.cocoChinaInfra` and then use a `me.showChinaVideo()` method to check it?

**Comment by shubhi1092** [comment]
In that case, I think we should use `cocoChinaInfra` instead of `chinaInfra`

File: app/models/User.coffee (hunk 1 above)
Comment (AndrewJakubowicz) [reply app/models/User.coffee]: Great point. Changing both side to reflect intention better.

**Pull request merged**